compiler/ballerina-backend-jvm-old/src/main/ballerina/compiler_backend_jvm/interop/external_method_gen.bal

```diff
@@ -66,6 +68,21 @@ function genJMethodForBExternalFuncOldStyle(OldStyleExternalFunctionWrapper extF
     TerminatorGenerator termGen = new(mv, indexMap, labelGen, errorGen, birModule);
     mv.visitCode();
 
+    jvm:Label? tryStart = ();
+    jvm:Label? tryEnd = ();
+    jvm:Label? tryHandler = ();
+    boolean isObserved = false;
+    if (isRemote) {
+        isObserved = true;
+        tryStart = labelGen.getLabel("try-start");
+        tryEnd = labelGen.getLabel("try-end");
+        tryHandler = labelGen.getLabel("try-handler");
+        if (tryStart is jvm:Label && tryEnd is jvm:Label && tryHandler is jvm:Label) {
```

Comment:
for each label, we can simply cast to `jvm:Label`... and get rid of the if condition..

Same applies in other places